src/proctored-exam-settings/ProctoredExamSettings.jsx

```diff
@@ -251,6 +274,37 @@ function ExamSettings(props) {
     );
   }
 
+  function renderSaveSuccess() {
+    const studioURL = StudioApiService.getStudioUrl(props.courseId);
+    return (
+      <Alert
+        variant="success"
+        dismissible
+        data-test-id="saveSuccess"
+        onClose={() => setSaveSuccess(false)}
+      >
+        Proctored exam settings saved successfully. You can go back to
+        <Alert.Link href={studioURL}>Studio here</Alert.Link>.
```

Comment:
Nit: Maybe indicate that this will take them back to their course - not just Studio?

```suggestion
        Proctored exam settings saved successfully. You can go back to your course in Studio
        <Alert.Link href={studioURL}>here</Alert.Link>.
```